src/gvmd.c

```diff
@@ -1106,11 +1106,15 @@ handle_sigabrt_simple (int signal)
 static int
 update_nvt_cache_osp (const gchar *update_socket)
 {
+#ifdef OPENVASD
+  setproctitle ("Openvasd: Updating NVT cache");
```

Comment:
Can we have a consistent naming? just openvasd. not OpenVASD and not Openvasd.
```suggestion
  setproctitle ("openvasd: Updating NVT cache");
```